### ISSUE TITLE
composer produces PHP error

### ISSUE THREAD

**Issue by james-nesbitt** [wunderkraut/wundertools-image-fuzzy-developershell#49]
When running composer in the shell cli, I get the following PHP error:

```
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php7/modules/.so' - Error loading shared library /usr/lib/php7/modules/.so: No such file or directory in Unknown on line 0
```

It is not clear if this problem is upstream.

**Comment by james-nesbitt** [comment]
In my case, the command is actually successful, but because of the bad php exension, the cli returns an error and the docker build aborts

**Comment by aleksijohansson** [comment]
The upstream was heavily updated so this image should be updated to use the latest upstream image as base

**Comment by aleksijohansson** [comment]
This issues has now been fixed in v7.0.12-0 of the upstream quay.io/wunder/alpine-php image.

**Comment by aleksijohansson** [comment]
Based to new upstream image so this is now fixed.
